Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqDataset.Tests.ps1

```diff
@@ -0,0 +1,85 @@
+﻿. $PSScriptRoot\..\BigQuery\GcbqCmdlets.ps1
+$project, $zone, $oldActiveConfig, $configName = Set-GCloudConfig
+
+Describe "New-GcbqDataset" {
+
+	It "should take a name, description, and time to make a dataset" {
+		$data = New-GcbqDataset "test_data_id1" -Name "Testdata" -Description "Some interesting data!" -Timeout 86400000
+		$data | Should Not BeNullOrEmpty
+		$data.DatasetReference.DatasetId | Should Be "test_data_id1"
+		$data.FriendlyName | Should Be "Testdata"
+		$data.Description | Should Be "Some interesting data!"
+		$data.DefaultTableExpirationMs | Should Be 86400000
+    }
+
+	It "should accept a dataset object from pipeline" {
+		$data = New-Object -TypeName Google.Apis.Bigquery.v2.Data.Dataset
+		$data.DatasetReference = New-Object -TypeName Google.Apis.Bigquery.v2.Data.DatasetReference
+        $data.DatasetReference.DatasetId = "test_data_id2"
+        $data.DatasetReference.ProjectId = $project
+		$data.FriendlyName = "PipeTest"
+		$data.Description = "Some cool data coming hot off the pipeline!"
+		$data | New-GcbqDataset -outvariable newdata
```

Comment:
Thanks for all of the comments.  Just fixed all of that, going to re-push.  

@ the additional tests, I'm planning to do another section below once I have all 4 Dataset Operations done that tests how they function as a group / as the developer would use them.